site/pages/rest-api/examples/update-analysis.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Update Analysis
+---
+
+Update a previously created Analysis.
```

Comment:
It might be a good idea to include a link create-analysis.md here. Also, the three .md files should be consistent in the (de)capitalization of "Analysis"